edison/static/js/login-signup.js

```diff
@@ -0,0 +1,81 @@
+var right_edge
+
+function ajaxCall(url, data, method, successCallback, errorCallback) {
```

Comment:
Wrapping the ajax call limits its functionallity.
If you would want to send an additional info with the ajax call, you will need to change the function signature and fix its call everywhere you used it.
This problem actually taking effect here because you should add contentType to the list of parameters ajax is taking.
The default contentType is "application/x-www-form-urlencoding", and we will need to send "application/json" as well.

Reply:
ack